Common/Systems/Synchronization/Handlers/SkillPassiveValueHandler.cs

```diff
@@ -37,9 +28,9 @@ private static void SetPlayerNodeStrength(byte player, string treeName, string n
 		Main.player[player].GetModPlayer<SkillTreePlayer>().ModifyPassive(SkillTree.TypeToSkillTree[tree.ParentSkill], nodeType, level, false, true);
 	}
 
-	internal override void ServerRecieve(BinaryReader reader)
+	internal override void ServerReceive(BinaryReader reader, byte sender)
 	{
-		ModPacket packet = Networking.GetPacket(MessageType);
+		ModPacket packet = Networking.GetPacket(Id);
 		byte target = reader.ReadByte();
```

Comment:
Excessive read on server.
```suggestion
		byte target = sender;
```